### DISCUSSION TITLE
Enable soft-reconfiguration inbound by default?

### DISCUSSION THREAD

**Discussion by jbemmel** [ipspace/netlab#1665] [Features]
When using a bgp lab for troubleshooting, it can be useful to see received routes. Any drawbacks/objections to enabling "soft-reconfiguration inbound" by default?

**Comment by ipspace** [comment]
> Any drawbacks/objections to enabling "soft-reconfiguration inbound" by default?

Yes. It is not needed now that most platforms support route refresh capability. If you want to have it, use a custom template.
